agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/README.md

```diff
@@ -0,0 +1,97 @@
+
+# AWS CDK (Python) to provision AWS Bedrock Agent with Knowledge Base and an Action Group Connection
+
+In this folder, we provide an example to create Bedrock Agent using Infrastructure as Code (IaC).IaC is implemented using with [AWS CDK Python APIs](https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_bedrock.html). The Bedrock agent is integrated with a Knowledge Base and an Action Group. With this integration, the agent will be able to respond to a user query by taking a sequence of actions, consulting the knowledge base to obtain more information, and/or executing tasks using the lambda function connected with an Action Group. 
+
+AWS CDK for Python is implemented with configurable parameters from 'config.json' file. 
+
+The Agent architecture can be referred [here](https://github.com/aws-samples/amazon-bedrock-samples/tree/main/agents-and-function-calling/bedrock-agents/features-examples/05-create-agent-with-knowledge-base-and-action-group)
+
+
+## Build and Deploy with AWS CDK 
+
+This project is set up like a standard Python project. The initialization process also creates a virtualenv within this project, stored under the .venv directory. To create the virtualenv it assumes that there is a python3 (or python for Windows) executable in your path with access to the venv package. If for any reason the automatic creation of the virtualenv fails, you can create the virtualenv manually.
+
+The `cdk.json` file tells the CDK toolkit how to execute your app.
+
+### Python Setup
+
+Manually create a virtualenv on MacOS and Linux:
+
+```
+python3 -m venv .env
+```
+
+After the bootstrap process completes and the virtualenv is created, use the following step to activate your virtualenv.
+
+```
+source .env/bin/activate
+```
+
+If you are a Windows platform, activate the virtualenv like this:
+
+```
+.env\Scripts\activate.bat
+```
+
+Once the virtualenv is activated, install the required dependencies.
+
+```
+pip3 install -r requirements.txt
```

Comment:
add the documentation to cd to the correct folder:` cd amazon-bedrock-samples/agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/`

Reply:
Added "cd amazon-bedrock-samples/agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/" as first command in python setup.